docs/documentation/advanced_examples/continuous-jump-measurement.md

```diff
@@ -0,0 +1,198 @@
+# Continuous jump measurement
+
+!!! Warning "Work in progress."
+    This tutorial is under construction, this is a draft version.
```

Comment:
We both know that we'll never come back to make it a not-draft version :D

So I'd propose removing this warning, and just accept that it's good enough for our standard of examples for now. And we can come back to it for improvements when we decide to generally improve the examples documentation.

Reply:
I think even if we never come back to it it's good to have different level of documentation readiness. Even for us, to keep track of which tutorials are drafty, which are not. To me it's rather a good way to ship tutorials before we are satisfied of how they look like. 😇